docker-compose.yml

```diff
@@ -93,3 +93,90 @@ services:
     environment:
       - SPRING_DATASOURCE_URL=jdbc:postgresql://db/kafka_pet
       - SPRING_KAFKA_BOOTSTRAP-SERVERS=broker:9092
+
+  schema-registry:
+    image: confluentinc/cp-schema-registry:6.2.10
+    environment:
+      - SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL=zookeeper:2181
+      - SCHEMA_REGISTRY_HOST_NAME=schema-registry
+      - SCHEMA_REGISTRY_LISTENERS=http://schema-registry:8081,http://localhost:8081
+    depends_on:
+      - broker
+      - zookeeper
+
+  kafka-connect:
+    image: confluentinc/cp-kafka-connect-base:6.2.0
```

Comment:
just a minor one, worth(but not mandatory) using the same version of connect cluster as a version of the broker.